### DISCUSSION TITLE
Database import Error

### DISCUSSION THREAD

**Discussion by hamedTrendlix** [planetscale/discussion#456] [Bug Reports] [answered]
![image](https://user-images.githubusercontent.com/120968501/235862499-5f412bbc-034c-4473-8c43-7d103a6a4980.png)

I am importing data from MariaDB on server and I got this error.

How can I solve it.

**Comment by jgreet** [comment, marked as answer]
Hi, have you seen our guide to MariaDB imports?

https://planetscale.com/docs/imports/mariadb-migration-guide

If the settings outlined there don't help, leave a failing import in place and [open a support ticket](https://support.planetscale.com/hc/en-us/requests/new). We can't import from every MariaDB database, PlanetScale and Vitess are built on MySQL, but we'll take a look.
